VideoLocker/src/main/java/org/edx/mobile/task/GetCommentsListTask.java

```diff
@@ -25,8 +25,15 @@ public GetCommentsListTask(@NonNull Context context, @NonNull String responseId,
     }
 
     public Page<DiscussionComment> call() throws Exception {
-        List<String> requestedFields = Collections.singletonList(
-                DiscussionRequestFields.PROFILE_IMAGE.getQueryParamValue());
+        final List<String> requestedFields;
+        {
```

Comment:
Ah well, this whole code is boiler plate, and I was expecting some suggestion regarding its refactoring, but I have 1 in mind now. Which'll rid of your concerns here too.